modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessor.java

```diff
@@ -274,4 +275,10 @@ protected ParameterValueResolver getParameterValueResolver() {
     final Event event = getInitialiserEvent(muleContext);
     return new OperationParameterValueResolver(new LazyExecutionContext<>(resolverSet, operationModel, extensionModel, event));
   }
+
+  @Override
+  public ProcessingType getProccesingType() {
+    return BLOCKING;
```

Comment:
this is just until there is non-blocking support in extensions, right? can you add a TODO?